### ISSUE TITLE
Transfer CWA to SD-Card

### ISSUE THREAD

**Issue by rawabe803** [corona-warn-app/cwa-wishlist#118]
There was a similar issue, but this was already marked as closed. So I open it again:

I would be very happy if there would be a possibility to move the CWA from internal memory to the SD-Card. This is due to the fact I own a smart phone with not so much internal memory (Samsung A3 (2017)), which I want to use some further years. Has something to do with sustainability. CWA runs fine on Android 8.

This feature is often rejected withconcerns about data security and/or possible malfunctions. Since I shift every app wherever possible to the SD card, I didn't experience any malfunction. The concern of data security could be encountered with encoding critical data, or keeping data in internal memory anyway.

CWA ist important to fight against Covid-19 an every opportunity should be taken to make it also working on smaller devices.

So please do me the favour and add this feature.

**Comment by SebastianWolf-SAP** [comment]
We already replied with an explanation in https://github.com/corona-warn-app/cwa-documentation/issues/114. The memory footprint of the app is really small and should be manageable on each device which supports the app. Moreover, there might be issues when apps are moved to an SD card. 

But OK, since then we introduced our cwa-wishlist repository. The issue is moved over and can be voted on.

Mit freundlichen Grüßen/Best regards, 
SW
Corona Warn-App Open Source Team
